.github/workflows/build.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
         pushd src
         ../utils/find_unused_includes.py
         popd
+    - name: Check regression test coverage of starting and win conditions
+      run: |
+        pip install iniparse
```

Comment:
iniparse can be replaced by configparser from core python. See comments in [utils/check_template_test_coverage.py](https://github.com/widelands/widelands/pull/6182/files#diff-8fd256175994905417783fda6ceb202031cea5eaca92ff8df52340ce29a388e1)